keps/sig-api-machinery/3352-aggregated-discovery/README.md

```diff
@@ -678,7 +679,12 @@ channel if you need any help or guidance. -->
 
 ###### Does enabling the feature change any default behavior?
 
-No
+Clients using client-go version 1.25 and up will use the aggregated
```

Comment:
Was this supposed to be 1.26 or did we backport the client changes from 1.26 to 1.25?  We don't normally backport features and I'm not clear on why this would be an exception.

Reply:
client-go 1.25 and apiserver 1.26. The 1.25 client checks whether aggregated discovery is enabled by the server and attempts to use aggregated document if available. We are going beta in 1.26 so 1.25 client with 1.25 apiserver will still use the old unaggregated discovery.

Reply:
We're fixing to build 1.27, right?  

I think the impl arrived roughly in Nov 2022 (https://github.com/kubernetes/kubernetes/pull/113599), so I think it landed in 1.26, but not 1.25.

Reply:
Yeah, I agree with David, we only started making changes in 1.26 IIRC?

Reply:
Ah you're right I got the release numbers completely mixed up. Thanks for catching!